draft-ietf-ccamp-dwdm-if-param-yang.txt

```diff
@@ -298,188 +304,44 @@ Internet-Draft   draft-ietf-ccamp-dwdm-if-param-yang-11        July 2024
    Curr-q-factor:
       "Q" factor estimated at Rx Transceiver port.
 
-4.2.  Use Cases
+3.4.  Use Cases
 
    The use cases are described in draft-ietf-ccamp-optical-impairment-
-   topology-yang (Section 2.5), draft-ietf-teas-actn-poi-applicability
-   and draft-poidt-ccamp-actn-poi-pluggable.
-
-4.3.  Optical Interface for external transponder in a WDM network
-
-   The ietf-ext-xponder-wdm-if is an augment to the ietf-interface.  It
-   allows the user to set the operating mode of transceivers as well as
-   other operational parameters.  The module provides also treshold
-   settings and notifications to supervise measured parameters and
-   notify the client.
-
-  module: ietf-ext-xponder-wdm-if
-
-    augment /if:interfaces:
-      +--rw optIf-templates
-         +--rw explicit-transceiver-modes
-            +--rw explicit-transceiver-mode*
-                    [explicit-transceiver-mode-id]
-               +--rw explicit-transceiver-mode-id        string
-               +--ro line-coding-bitrate?                identityref
-               +--ro bitrate?                            uint16
-               +--ro max-diff-group-delay?               decimal-2
-               +--ro max-chromatic-dispersion?           decimal-2
-               +--ro cd-penalty* [cd-value]
-               |  +--ro cd-value         decimal-2
-               |  +--ro penalty-value    union
-               +--ro max-polarization-mode-dispersion?   decimal-2
-               +--ro pmd-penalty* [pmd-value]
-               |  +--ro pmd-value        decimal-2
-
-
+   topology-yang, draft-ietf-teas-actn-poi-applicability and draft-
+   poidt-ccamp-actn-poi-pluggable-usecases-gaps.
 
-Galimberti, et al.       Expires 4 January 2025                 [Page 6]
-
-Internet-Draft   draft-ietf-ccamp-dwdm-if-param-yang-11        July 2024
+4.  Definitions Section
 
+   [Editor Note: Changed the section title according to draft-ietf-
+   netmod-rfc8407bis]
 
```

Comment:
draft-ietf-netmod-rfc8407bis requires to have one or more definition sections not to have a section with the tile "Definition Section"

```suggestion
```